### ISSUE TITLE
[ENH] - Provide A Quick Edit App

### ISSUE THREAD

**Issue by grover556** [NAPSG/SARCOP#304]
### Which app?

Intel Manager 2.0

### Describe requirements

Intel Manager 1.0, 2.0 and Admin Editor are very heavy pages that require tons of cloud resources on an often restricted or strained internet connection. The initial moments when a team is tasked to create search grids and edit points is normally when internet access is the worst. IM 1/2 and AE all load tons of layers that takes bandwidth, and even if they are set to hidden, the Javascript still has to download layer definitions and capabilities for each layer. As it stands today, 13 MB are transferred when clicking on Intel Manager 2.0.

Add a Quick Edit tab that is separate from the Intel Manager dropdown tabs. This Quick Edit should only contain bare minimum functions for a bandwidth restricted environment:
- Waypoint, Tracklog, Segment layers (maybe Divisions too)
- Allow filtering based on: team, datetime, waypoint type, waypoint follow-up status, tracklog type, segment type, segment status
- Smart Editor to add/edit waypoint, tracklog, segment layers
- Print widget

### Alternatives

I spent a large amount of time this past time out adding/editing segments and points in Field Maps on my phone because it was easier than waiting for the Dashboards to load.

### Priority impact

p0 - not time sensitive

**Comment by pjdohertygis** [comment]
Consider an Intel Manager "Light" or a programmatic way to create team focused web maps...

**Comment by pjdohertygis** [comment]
Let's consider a simple instant app or web map that you can link out to, test if it loads fewer resources. @grover556 are you willing to help us test this?

**Comment by pjdohertygis** [comment]
A quick test of an Instant App vs Intel Manager 2.0 does not show much difference if using the same web map @grover556 

Intel Manager 2.0 608kb transferred 51MB resources
![image](https://github.com/user-attachments/assets/6f7fa5d0-57f9-4c41-b0d4-26edabb95d35)

Instant App 3.8MB transferred, 52 MB resources
https://fema.maps.arcgis.com/apps/instant/sidebar/index.html?appid=30aea0a0a57345d3a293a6f7601a8eb6
![image](https://github.com/user-attachments/assets/0af349b9-5005-45ec-be52-643d3c1c0dae)

The next test would be to strip down the web map and try a few other instant app options.

**Comment by pjdohertygis** [comment]
Here is a test of the Manager Instant App 

![image](https://github.com/user-attachments/assets/3598dbb5-bf07-4bff-834a-437b6262525e)

One advantage is the Filter button and the mixed map / table interface for editing. 
![image](https://github.com/user-attachments/assets/09a0d551-2ae8-445f-9d50-ca6ed9b3519f)

**Comment by pjdohertygis** [comment]
Here is the same light Sandbox web map in the "Manager" Instant App - in theory, this is FASTER than the web map on it's own. 
https://napsg.maps.arcgis.com/apps/instant/manager/index.html?appid=6e0df5b2d86d49029f864715d787fd00

641 requests
2.3 MB transferred
31.5 MB resources
Finish: 10.68 s
DOMContentLoaded: 918 ms

![image](https://github.com/user-attachments/assets/4cbbe5cf-52ea-4707-93f7-ef4fcfc4286d)

**Comment by pjdohertygis** [comment]
Here is the same web map in the "Sidebar" Instant App - so far this is the fastest option.
https://napsg.maps.arcgis.com/apps/instant/sidebar/index.html?appid=9e38f8700195486685b298b648ab560c

754 requests
2.3 MB transferred
46.7 MB resources
Finish: 7.57 s
DOMContentLoaded: 359 ms

This option would be worth exploring because you could potentially "pre-filter" the app for your Team. 
![image](https://github.com/user-attachments/assets/862b3e45-b6ac-4833-96da-89d40b27d912)

**Comment by pjdohertygis** [comment]
@kstrube we can keep this on the v10 project, but I might ask you to get input from the ArcGIS Online Team. What is the "lightest" Esri app? See my limited testing above.

I know we could probably host something custom with Leaflet for fast display, but don't think the API has an editing capability. https://developers.arcgis.com/esri-leaflet/

@grover556 you still happy to help us with testing this as we get closer?

**Comment by grover556** [comment]
@pjdohertygis yes I am happy to help test any potential lightweight tools. If we've got a place to host a custom JavaScript API project, I'm pretty proficient in that and have made similar tools using that framework.
